instrumentation/grape/lib/opentelemetry/instrumentation/grape/event_handler.rb

```diff
@@ -96,7 +96,7 @@ def path(endpoint)
             version = endpoint.routes.first.options[:version] || ''
             prefix = endpoint.routes.first.options[:prefix]&.to_s || ''
             parts = [prefix, version] + namespace.split('/') + endpoint.options[:path]
-            parts.reject { |p| p.blank? || p.eql?('/') }.join('/').prepend('/')
+            parts.reject { |p| p.nil? || p.empty? || p.eql?('/') }.join('/').prepend('/')
```

Comment:
Would safe navigation work?
```suggestion
            parts.reject { |p| p&.empty? || p.eql?('/') }.join('/').prepend('/')
```

Reply:
In this case, no, because `p&.empty?` returns `nil` if `p` is `nil`, which evaluates to `false` as a boolean.

```ruby
3.1.3 :001 > p = nil
 => nil
3.1.3 :002 > p&.empty?
 => nil
3.1.3 :003 > p.nil? || p.empty? || p.eql?('/')
 => true
3.1.3 :004 > p&.empty? || p.eql?('/')
 => false
```

I guess there are refactor alternatives, but even though it is a bit verbose I thought it was quite readable like this. But of course I'm open to other suggestions 😃